bigtable/admin.go

```diff
@@ -40,13 +40,22 @@ import (
 	gtransport "google.golang.org/api/transport/grpc"
 	btapb "google.golang.org/genproto/googleapis/bigtable/admin/v2"
 	"google.golang.org/genproto/protobuf/field_mask"
-	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/metadata"
-	"google.golang.org/grpc/status"
 )
 
 const adminAddr = "bigtableadmin.googleapis.com:443"
 
+// ErrPartiallyUnavailable is returned when some locations (clusters) are
+// unavailable. Both partial results (retrieved from available locations)
+// and the error are returned when this exception occurred.
+type ErrPartiallyUnavailable struct {
```

Comment:
In the Java client, the exception holds unavailable zones and instances and can be inspected to retrieve the results. Can we do something similar here? Example: https://github.com/googleapis/java-bigtable/blob/master/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/PartialListInstancesException.java

Reply:
In case of some locations are unavailable, we'll return an error (which includes unavailable locations) and the results (those are available) in the same time. So it supposed to work this way already.